Source/Core/DolphinWX/LogWindow.cpp

```diff
@@ -359,5 +361,13 @@ void CLogWindow::Log(LogTypes::LOG_LEVELS level, const char* text)
   if (msgQueue.size() >= MSGQUEUE_MAX_SIZE)
     msgQueue.pop();
 
-  msgQueue.emplace(static_cast<u8>(level), StrToWxStr(text));
+  // Convert from UTF-8
+  wxString encoded_message = StrToWxStr(text);
+  // Fallback to OS locale if conversion failed
+  if (encoded_message.empty())
+  {
+    PanicAlert(StringFromFormat("Conversion from UTF-8 failed with: \"%s\"", text).c_str());
```

Comment:
Won't this be an issue for SHIFT-JS encoding and other encodings? We'll loose the message and potentially where it's located so it's unhelpful unless a debugger is attached.